estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import models, fields
+
+class estate_property(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+
+    name= fields.Char("Plan Name", required = True ) # required make property not nullable 
```

Comment:
```suggestion
    name = fields.Char("Plan Name", required=True ) # required make property not nullable 
```
spacing must be same through the application.